libraries/device-registry/CMakeLists.txt

```
@@ -10,4 +13,8 @@ target_link_libraries(gz-sim-yarp-device-registry
     YARP::YARP_os
     YARP::YARP_init)
 
-install(TARGETS gz-sim-yarp-device-registry)
+install(TARGETS gz-sim-yarp-device-registry
+        EXPORT ${PROJECT_NAME})
+
+install(FILES DeviceRegistry.hh 
+        DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/gz-sim-yarp-device-registry")
```

As a matter of taste, I like to keep the same include directory for all headers that the project will install. Furthermore, it would be nice to use the string used by the C++ namespace, perhaps it could make sense to use `gzyarp` also as the directory where we install the headers?

```suggestion
        DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/gz-sim-yarp-device-registry")
```

---

Thanks!